prog/dftb+/lib_dftb/scc.F90

```diff
@@ -418,6 +431,16 @@ subroutine Scc_initialize(this, env, inp)
     this%tDampedShort(:) = inp%tDampedShort(:)
     this%dampExp = inp%dampExp
 
+    ! H5 correction
+    ! Input data are saved in the SCC object
+    this%useH5 = inp%useH5
```

Comment:
Could instead use allocated(inp%h5Correction) to signal this.

Reply:
I've switched to allocated(inp%h5Correction) in the input data module as suggested. However, I still keep a separate flag for it in the scc module for performance reason, as it is evaluated repeatedly.